### ISSUE TITLE
[BUG] ZOTEROITEMCOLLECTIONS doesn't return correct output

### ISSUE THREAD

**Issue by mdroidian** [alixlahuec/zotero-roam#108]
**zoteroRoam version :**
v0.7.7 (f91980f6292207d0d8d0dd61adb1b6bdcf85e363)
Roam Depot Version: 2

**What happened ?**

Add SmartBlock with
```
- <%ZOTEROITEMCOLLECTIONS:true%>
- <%ZOTEROITEMCOLLECTIONS:false%>
- <%ZOTEROITEMCOLLECTIONS%>
```
click `Add metadata` on roam page

Get 
`Block threw an error while running: <%ZOTEROITEMCOLLECTIONS:true%>` on page 
and 
`TypeError: window.zoteroRoam.getItemCollections(...).map is not a function` in console

**To Reproduce**
https://user-images.githubusercontent.com/3792666/204112897-8b0f27d6-5c41-4fd6-adba-aba632eedd5a.mp4

**Expected behavior**

To get item collections returned

---

**System :**
[x] Windows
[  ] Mac
[  ] Linux
[  ] Android
[  ] iOS

**Context :**
[x ] Browser, specify : Chrome Version 107.0.5304.121
[  ] Roam desktop app
[  ] Roam mobile PWA

**Screenshots**
If applicable, add screenshots and console logs to explain & help investigate the problem.

![image](https://user-images.githubusercontent.com/3792666/204112935-80cfd68f-f7b2-425e-b91d-5eb962d0e0c1.png)

**Additional information**

**Comment by alixlahuec** [comment]
Thanks for flagging that -- problem is a typo here: https://github.com/alixlahuec/zotero-roam/blob/b0036f3c7c6caf54cc387ea85c5e282fc0f45405/src/smartblocks.js#L39

Will fix asap!

**Comment by alixlahuec** [comment]
The fix just got released to Roam Depot (v0.7.8) 🚀
